events/app/views/pages/events.njk

```diff
@@ -15,8 +15,9 @@
 {% endblock %}
 
 {% block body %}
+  {% componentV2 'list-header', {name: 'Events', description: 'Past and present events at Wellcome Collection', total: eventsList.results.length} %}
```

Comment:
Strange self-explanatory sub-header?
Might be worth syncing up with the page description that would be better for SEO that humans?